Themes/default/PersonalMessage.template.php

```diff
@@ -61,7 +61,7 @@ function template_pm_popup()
 	echo '
 		<div class="pm_bar">
 			<div class="pm_sending block">
-				', $context['can_send_pm'] ? '<a href="' . $scripturl . '?action=pm;sa=send">' . $txt['pm_new_short'] . '</a> | ' : '', '
+				', $context['can_send_pm'] ? '<a href="' . $scripturl . '?action=pm;sa=send">' . $txt['pm_new_short'] . '</a>' . ($context['can_draft'] ? ' | ' : '') : '', '
```

Comment:
Why not combine with the below ternary?

Reply:
For some reason I was thinking that $context['can_draft'] could be true when $context['can_send_pm'] is false. Will fix it shortly

Reply:
oh your right.  Logic does currently allow that.